### ISSUE TITLE
Ideas

### ISSUE THREAD

**Issue by cecton** [rustminded/recruit-rust#3]
- [ ] list of techs like list of tags mixing languages and frameworks
- [ ] techs can be "certified" with public contributions and OSS
- [ ] list of pro experience: company name, website, job description and list of techs
- [ ] list of public/oss contributions: each item include a list of tech
- [ ] list of personal projects: each item include a list of tech
- [ ] candidates has timezones (multiple if they want)
- [ ] search can filter per timezones (sort also by closer)
- [ ] search can filter by tech (sort by most matching techs)
- [ ] focus on remote working
- [ ] on candidate page, clicking on a tech highlight all occurrences of the tech tag  in the page
- [ ] license to gplv3 to avoid commercialization
- [ ] donation link
- [ ] local storage: categorize candidates: selected/pending/unselected (pending by default) /archive (not included in listing)
- [ ] filter on selected
- [ ] easy print candidates (pdf?)
- [ ] contract type: employee/contractor
- [ ] contractor can use a branch
- [ ] employees must use revision
- [ ] employees are limited in time on the site
- [ ] generate content at compile time
- [ ] field availability: full time, part time, not available (hidden from the system)
- [ ] secondary techs used for filtering: react implies js, js implies react and angular, docker implies container. searching a tech brings secondary techs but with lower priority
- [ ] candidates make crates (unpublished) to make their entry. then they make a pr to add the crate
- [ ] certificates?
- [ ] if you click on the tech in the list, it opens the profile and highlight the tech
- [ ] print listing of candidates
- [ ] only when print: area for the recruiter to take notes
- [ ] button to clear the selected candidates
